lessons/AI/ai-statistical-lang-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,452 @@
+@A{Yes! With statistical language modeling, AI can compose music, play chess games, and more. The "text" need not be made up of words: any symbolic notation at all will do.}
+}
+
+
+
+== Constructing a Statistical Language Model
+
+@objective{slm}
+
+=== Overview
+
+Students construct a statistical language model by decomposing the text and computing the probabilities of different words appearing.
+
+=== Launch
+
+The best way to make sense of statistical language modeling is to try it yourself! We'll start by constructing a model.
+
+For our corpus, we will use the folk song @handout{old-lady-lyrics.adoc, "There Was an Old Lady Who Swallowed a Fly"}, which tells the nonsensical story of an old lady who swallows a fly, and the unfortunate series of events that follows.
+
+First, we will decompose the title of our corpus into differently sized chunks (one word at a time, two words at a time, etc.):
+
+[cols="^.^1,^.^1,<.^8", stripes="none", options="header"]
+|===
+
+| chunk size | Quantity			| Decomposition
+
+| 1 word
+| 9
+| (There) (Was) (an) (Old) (Lady) (Who) (Swallowed) (a) (Fly)
+
+| 2 words
+| 8
+| (There Was) (Was an) (an Old) (Old Lady) (Lady Who) (Who Swallowed) (Swallowed a) (a Fly)
+
+| 3 words
+| 7
+| (There Was an) (Was an Old) (an Old Lady) (Old Lady Who) (Lady Who Swallowed) (Who Swallowed a) (Swallowed a Fly)
+
+|===
+
+The formal word computer scientists use in this context is not "chunk" but @vocab{n-gram}. In an @math{n}-gram, @math{n} represents the number of words in the chunk. For special cases where @math{n} is 1, 2, or 3, the @math{n}-grams are called unigrams, bigrams, and trigrams.
+
+
+=== Investigate
+
+Let's dig a little deeper...
+
+@teacher{
+To share the song lyrics with students,  have students @handout{old-lady-lyrics.adoc, "read them independently"}. If desired, you could also listen to a recorded version of the song.
+}
+
+The phrase "there was an old lady who swallowed a..." is repeated in our corpus! Let's zoom in on one unigram from that phrase: “there”.
+
+@QandA{
+@Q{Referring to the @handout{old-lady-lyrics.adoc, "lyrics"}: how many times does the word "there" appear in the song?}
+@A{4}
+@Q{In this corpus, how many times was the word "there" followed by the word "was"?}
+@A{4}
+@Q{What is the probability that the word "there" is followed by the word "was"?}
+@A{4/4 or 100%}
+}
+
+In the example you just worked through, you computed the probability that "was" appears after the unigram "there". We can represent the computation you just completed with a special notation:
+
+@math{p(was | there) =}
+@math{\frac
+	{\textit{count(there was)}}
+	{\textit{count(there...)}}
+= {\frac{4}{4}}}
+
+Put another way: To compute the probability that "was" follows "there", we divide 4 (how many times we see "was" follow "there") by 4 (how many times we see "there" followed by anything).
+
+
+@lesson-instruction{
+- Complete @printable-exercise{constructing-model.adoc}.
+}
+
```

Comment:
What are you hoping they learned from this activity? It feels like there should be a QandA here to get at that.